lib/fizzbuzz.rb

```diff
@@ -3,6 +3,15 @@ class SuperFizzBuzz
   def run(input)
 
     #Implement your code here
+    if input % 3 == 0 && input % 5 == 0
```

Comment:
I think you can make this much simpler with some guard clauses and a running tally. I actually think you could get this down to 4 lines without repeating your self on the modular checks!